docs/blog/20230701-Daeraxa-JulyUpdate.md

```diff
@@ -0,0 +1,127 @@
+
+This work is still very much ongoing and we have new grammars being migrated all the time - our latest is the [TOML Tree-sitter grammar](https://github.com/pulsar-edit/pulsar/pull/617) that [@savetheclocktower] has added which come with all the benefits of a Tree-sitter grammar such as more consistent highlighting and scope for new features in the future.
+
+You can try this out in our [rolling release](https://pulsar-edit.dev/download.html#rolling-release) or wait until the next regular one. Either way you will need to tick the `Use Modern Tree-Sitter Implementation` option in your `Core` settings in order to enable it.
+
+As ever we are after feedback, so if you use this new TOML Tree-sitter grammar (or any other for that matter) and have any issues or see any inconsistencies compared to the existing grammar then please let us know on [Discord](https://discord.gg/7aEbB9dGRT) or [file an issue](https://github.com/pulsar-edit/pulsar/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).
+
+## `less-cache` package update
+
+<img src="./assets/less.png" height=200>[^4]
+
+This is still in the works but [@confused-techie] has been making some updates to be added in the near future that bumps the version of `less-cache` that we are using.
+
+`less-cache` is a module that handles turning all Pulsar's (and our community's) packages written in [less](https://lesscss.org/) into valid CSS, as well as ensuring each one is able to import the values from Pulsar that it cares about. For the past three years `less-cache` has been using `less@3.12.2` which, while fine, has been missing out on some of the big new changes in less after version 4 was released.
+So [this PR](https://github.com/pulsar-edit/pulsar/pull/611) bumps it to `less-cache@2.0.0` in order to bump the version of `less` to 4.1.3 which brings along these exciting new features but also introduces breaking changes to existing `less` stylesheets. The biggest of which being:
+
+- Parantheses required for mixin calls
+- Parens-division now the default math setting
+
+The most noticeable thing, at least within Pulsar, is that all division is required, and all calculations are recommended, to be enclosed within parentheses. Essentially meaning `less` does math less eagerly, as opposed to 3.x which is described as doing math eagerly.
+
+So existing stylesheets in syntax themes, UI themes and community packages need to be updated to ensure that `less` properly does the math rather than handing off invalid CSS, with the math still included. At times an error may be shown informing the user the stylesheet failed to be compiled, whereas in some cases, it won't be reported and instead can result in a broken UI. Pulsar itself has already been updated, as well as all core packages, but now we have to keep a lookout for non-archived community packages that may need help with this change.
```

Comment:
```suggestion
We're brainstorming ways that we can detect these usages and minimize the impact on these packages. But if you've got a community package that you're maintaining, and it uses`less` stylesheets, please take a moment to see if it's affected. At times an error may be shown informing the user that the stylesheet failed to be compiled; in other cases, it won't be reported and instead can result in a broken UI.
```

Reply:
Thanks for adding this! I know this is a newer development, that was thought of after the original thing was written, so appreciate it!